multisig-transactions-monitor/package.json

```
@@ -21,14 +21,14 @@
     "test": "jest"
   },
   "dependencies": {
-    "forta-agent": "^0.1.1",
-    "bignumber.js": "^9.0.1"
+    "bignumber.js": "^9.0.1",
+    "forta-agent": "^0.1.1"
   },
   "devDependencies": {
     "eslint": "^8.10.0",
-    "eslint-plugin-import": "^2.25.4",
     "eslint-config-airbnb": "^19.0.4",
```

```suggestion
    "eslint-config-airbnb-base": "^15.0.0",
```

Suggesting this for consistency with all the other bots